edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Mqtt/DeviceProxy.cs

```diff
@@ -164,7 +164,7 @@ static class Metrics
             static readonly IMetricsCounter SentMessagesCounter = Util.Metrics.Metrics.Instance.CreateCounter(
                 "messages_sent",
                 "Messages sent from edge hub",
-                new List<string> { "from", "to", "from_route_output", "to_route_input" });
+                new List<string> { "from", "to", "from_route_output", "to_route_input", "priority" });
```

Comment:
Do you need this for device proxy? Don't priorities only apply to upstream messages? For local messages I think the tag should just be left off.

Reply:
it's primarily used for upstream, but priority works for both cloud endpoint and brokered endpoint.

---
In reply to: [397351570](https://github.com/Azure/iotedge/pull/2730#discussion_r397351570) [](ancestors = 397351570)